HunterPie.UI/Overlay/Widgets/Player/Views/PlayerHudView.xaml

```diff
@@ -40,11 +41,19 @@
         </Grid.ColumnDefinitions>
         
         <!-- Player name -->
+        <Image d:Source="{StaticResource ICON_BATTLE_EYE}"
+               Source="{Binding CombatStatus, Mode=OneWay, Converter={StaticResource CombatStatusToIconConverter}}"
+               Height="20"
+               Width="20"
+               VerticalAlignment="Center"
+               HorizontalAlignment="Left"
+               Margin="5,5,0,0"
+               Grid.Column="1"/>
         <TextBlock FontFamily="{StaticResource HUNTERPIE_DEFAULT_FONT}"
                    Foreground="{StaticResource WHITE_100}"
                    VerticalAlignment="Center"
                    HorizontalAlignment="Left"
-                   Margin="5,5,0,0" 
+                   Margin="25,5,0,0" 
                    Grid.Column="1">
             <Run Text="Lv."
                  Foreground="{StaticResource WHITE_400}"
```

Comment:
This wouldn't be an issue if the `Image` was always there, but since it's dynamic and when the player is not in combat the image won't show anything, just an empty space, I think it would be weird for the player name to have a big spacing on the left side. You can use a `StackPanel` and put both the `Image` and Player's name inside it so it will automatically handle this spacing.
You'll also need to add a `CombatStatusToVisibilityConverter` to make it so the Image is `Collapsed` when the status is `None`.

Reply:
Hmm, No icon displayed when out of combat.
Wouldn't it be a bit odd that the position of the player name keeps changing?
Is it okay because it doesn't happen often?

Reply:
Hmm, true, it would be annoying to see the name moving left and right everytime, in that case I can see two possible solutions:
1. Have an icon for when the player is not in combat too, maybe a greyed-out icon
2. Put the icon somewhere else, where it doesn't affect the position of other visual elements.

Reply:
![combatstatus](https://user-images.githubusercontent.com/50101064/208975010-f41bc4e9-740a-4e11-8083-d1c60c7af567.png)

It placed static position.

Reply:
Hmm, it looks off. I'll take a look at the widget design tomorrow and see a proper place for that icon.

Reply:
Honestly I've been trying to find a good place for the icon and I don't think it fits anywhere in the player's hud. Maybe a shadow on the icon diamond but not a whole icon for it.